templates/search_page.html

```diff
@@ -36,22 +37,37 @@ <h1 style="color: #24c1e0; text-align: center; background-color: whitesmoke">
     Vacation Planner
 </h1>
 
+<!--forms and inputs-->
 <div class="container" style="align-items: center">
```

Comment:
May consider adding some margin between this div.container and the above h1 header. 
For example, `<div class="container mt-3" style="align-items: center"> `
`mt-3` adds margin on the top of this div.

Reply:
Done.